packages/ra-ui-materialui/src/button/SaveButton.tsx

```diff
@@ -99,20 +114,11 @@ export const SaveButton = <RecordType extends RaRecord = any>(
                 // this button doesn't submit the form, so it doesn't trigger useIsFormInvalid in <FormContent>
                 // therefore we need to check for errors manually
                 event.preventDefault();
-                const isFormValid = await form.trigger();
-                if (!isFormValid) {
-                    event.preventDefault();
-                    notify('ra.message.invalid_form', { type: 'warning' });
```

Comment:
I no longer see this notification in the new code

Reply:
Because the `form.handleSubmit(handleSubmit)(event);` call already shows the same notification when needed